packages/gafl-webapp-service/src/processors/__tests__/recurring-pay-reminder-display.spec.js

```diff
@@ -0,0 +1,25 @@
+import { recurringPayReminderDisplay } from '../recurring-pay-reminder-display.js'
+
+const getCatalog = () => ({
+  recurring_payment_set_up_bulletpoint_4_email: 'we will send you an email showing the cost before the next payment is taken',
+  recurring_payment_set_up_bulletpoint_4_letter: 'we will send you a letter showing the cost before the next payment is taken',
+  recurring_payment_set_up_bulletpoint_4_text: 'we will send you a text showing the cost before the next payment is taken'
+})
+
+const getPermission = (reminder) => ({
+  licensee: {
+    preferredMethodOfReminder: reminder
+  }
+})
+
+describe('recurringPayReminderDisplay', () => {
+  it.each([
+    ['Email', 'we will send you an email showing the cost before the next payment is taken'],
+    ['Letter', 'we will send you a letter showing the cost before the next payment is taken'],
+    ['Text', 'we will send you a text showing the cost before the next payment is taken']
+  ])('returns correct wording for reminder', (reminder, expected) => {
```

Comment:
Identical name for three tests - if one of them fails, how will we know which one it is? Please differentiate them by using parameters